### PR TITLE
Switched to async transactions + change in `convert_cards` behavior.

### PR DESCRIPTION
**Pull request by mzhurovich**
@dkorolev PTAL

### CODE REVIEW

File: server.h [hunk 1]
```diff
@@ -91,65 +84,67 @@ class CTFOServer {
...
+                                  device_id.c_str(),
+                                  app_key.c_str(),
+                                  token.c_str()));
+
+                CopyUserInfoToResponseEntry(user, user_entry);
+                user_entry.token = token;
+
+                ResponseFeed rfeed;
+                GenerateResponseFeed(data, user_entry, feed_count, rfeed);
+                return Response(rfeed);
```
Comment (dkorolev) [inline server.h]: `return Response(rfeed, "feed");` ?

File: server.h [hunk 2]
```diff
@@ -167,93 +162,43 @@ class CTFOServer {
...
+                } else {
+                  DebugPrint(
+                      Printf("[/ctfo/feed] Token validated. Requested URL = '%s'", requested_url.c_str()));
+                  const auto user = data.Get(uid);
+                  ResponseUserEntry user_entry;
+                  CopyUserInfoToResponseEntry(user, user_entry);
+                  user_entry.token = token;
+                  ResponseFeed rfeed;
+                  GenerateResponseFeed(data, user_entry, feed_count, rfeed);
+                  return Response(rfeed);
```
Comment (dkorolev) [inline server.h]: `return Response(rfeed, "feed");` ?

File: server.h [hunk 3]
```diff
@@ -290,6 +235,57 @@ class CTFOServer {
     }
   }
 
+  void GenerateResponseFeed(StorageAPI::T_DATA& data,
```
Comment (dkorolev) [inline server.h]: s/`void`/`ResponseFeed`/ ?

**Comment by dkorolev** [comment]
@mzhurovich LGTM!

**Comment by mzhurovich** [comment]
@dkorolev fixed according to the comments.

**Pull request merged**